src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -159,4 +159,58 @@
         <Setter Property="Width" Value="{StaticResource StandardBoxMinWidth}" />
         <Setter Property="HorizontalAlignment" Value="Left" />
     </Style>
+    <!-- DELETE BUTTON STYLING IN COLORSCHEMES.XAML FILE -->
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Light">
+            <Style x:Key="SecondaryTextBlockStyle" TargetType="TextBlock">

```

Comment:
You cannot use the `x:Key="SecondaryTextBlockStyle"` key here because I already used this key above for a different purpose. You need to name this style differently here like calling it `x:Key="DeleteButtonStyle"` here and below in the other theme dictionaries.